pkg/model/storage/indexation.go

```diff
@@ -41,11 +42,11 @@ func (i *Indexation) GetMessageID() hornet.MessageID {
 // ObjectStorage interface
 
 func (i *Indexation) Update(_ objectstorage.StorableObject) {
-	panic(fmt.Sprintf("Indexation should never be updated: %v, MessageID: %v", i.indexationHash.ToHex(), i.messageID.ToHex()))
+	panic(fmt.Sprintf("Indexation should never be updated: %v, MessageID: %v", i.Index, i.messageID.ToHex()))
```

Comment:
Should we print the bytes array here or the "string()"? :shrug:

Reply:
gonna print hex